src/main/resources/view/HelpWindow.css

```diff
@@ -16,4 +16,5 @@
 
 #helpMessageContainer {
     -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-padding: 10; /* Adds padding of 10 to all sides */
```

Comment:
Nice update to the padding but maybe can also update the screenshot of the help window to the user guide

Reply:
That's a good idea